### ISSUE TITLE
Accessibility requirements

### ISSUE THREAD

**Issue by andygnewman** [Financial-Times/n-teaser#49]
cc: @i-like-robots 

Some elements are not semantically helpful to screen readers, especially the meta information (tag and prefix).

These should have aria-labels added.
